src/plays/image-gallery/Gallery.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import GalleryAPI from './GalleryAPI'
+
+
+export default function Gallery() {
+	const [data, setData] = useState(GalleryAPI)
+	const [ourText, setOurText] = useState("Welcome to this Play, Click on the Images and get to know about it.")
+	const [midText, setMidText] = useState("Sorry for interrupting. I think you like this play. Don't forget to give it a thumbs up.")
+	const msg = new SpeechSynthesisUtterance()
+	var voiceName = "Zira"
```

Comment:
Are there other voice types? Would it be cool to allow users to select one with Zira being the default?

Reply:
Means the user can select different voice types?

Reply:
Yes, just a suggestion. We can do it as a version 2 later as well. btw.. let me know when you have taken care of the comments/

@Shivam-Katare

Reply:
Okay, then I will resolve all the issues and let you know after that.